pkg/controller/operators/openshift/clusteroperator_controller.go

```diff
@@ -0,0 +1,247 @@
+package openshift
```

Comment:
Taking a brief look and my thought is, should this just be a downstream PR and not merged into this repository? Or is the plan to merge, sync, and then take this out. This is the most openshift specific code I've ever seen proposed to be checked into this repo. 

I know this is a bit annoying for you as the author, as we have to iron the split out, but I'm just wondering.

Reply:
it needs to be backported to release-4.6, which we're not capable of doing completely downstream-only yet.